DOCKER.md

```diff
@@ -0,0 +1,70 @@
+# Using Vectara Answer with Docker
+
+## Introduction
+
+To demonstrate how to use Vectara Answer with Docker we will now walk through the steps for building an application with the content of Paul Graham's essays website (http://paulgraham.com/). 
```

Comment:
Would it be worth adding a couple sentences here about why you'd want to use Docker? For example, does it help with deployment?